pkg/dbg/dbg.go

```diff
@@ -0,0 +1,96 @@
+package dbg
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/signal"
+	"runtime"
+	"runtime/pprof"
+	"syscall"
+	"time"
+
```

Comment:
nit: remove empty line number 12 so imports for formatted.

Reply:
Done